test/unit/bind-radio-request.spec.js

```diff
@@ -76,14 +76,14 @@ describe('Marionette.bindRadioRequests', function() {
   describe('when bindings is not an object', function() {
     beforeEach(function() {
       this.run = function() {
-        Marionette.bindRadioRequests(this.target, this.channel, 'replyFooStub');
+        Marionette.bindRequests(this.target, this.channel, 'replyFooStub');
       }.bind(this);
     });
 
     it('should error', function() {
       expect(this.run).to.throw(Marionette.Error, new Marionette.Error({
         message: 'Bindings must be an object.',
-        url: 'marionette.functions.html#marionettebindradiorequests'
+        url: 'marionette.functions.html#marionettebindRequests'
```

Comment:
What do you think if I change doc for `bindEvents` but for others in scope of task #3055 ?